crates/hyperswitch_connectors/src/connectors/worldpay.rs

```diff
@@ -617,7 +617,7 @@ impl ConnectorIntegration<Capture, PaymentsCaptureData, PaymentsResponseData> fo
                         .map(|id| id.to_string())
                 });
                 Ok(PaymentsCaptureRouterData {
-                    status: enums::AttemptStatus::Pending,
+                    status: enums::AttemptStatus::Charged,
```

Comment:
What are failure scenario's in capture flow ?

Reply:
Using the AttemptStatus transformation here now